packages/core/src/amazonq/webview/ui/connector.ts

```diff
@@ -348,4 +361,17 @@ export class Connector {
                 break
         }
     }
+
+    onCustomFormAction = (
+        tabId: string,
+        messageId: string | undefined,
+        action: any,
+        eventId: string | undefined = undefined
+    ): void | undefined => {
+        switch (this.tabsStorage.getTab(tabId)?.type) {
+            case 'gumby':
+                this.gumbyChatConnector.onCustomFormAction(tabId, action)
+                break
+        }
```

Comment:
nit: Will we have >1 entry in this switch going forward? Otherwise maybe convert to if?

Reply:
I could see other apps needing to use buttons in the future, so i'll keep this as a switch